### PR TITLE
The EditText in Payments dialog has been changed to TextInputLayout t…

### LINKED ISSUE #756: TextInputLayout in set amount dialog box
**Summary:**

The Set Amount alert dialog box uses `EditText` while at all the other places `TextInputLayout` has been used. This breaks the consistency in the app user interface

**Steps to reproduce:**

*Go to Payments
* Tap on Show code
* Tap on Set Amount 

**Expected behavior:**

`TextInputLayout` should have been used to maintain consistency in app's user interface

**Screenshots:**

WhatsApp Image 2020-02-24 at 9 25 57 PM

### PR DESCRIPTION
**Pull request by haran2248**
…o improve Interface.
WhatsApp Image 2020-07-30 at 12 27 09

## Issue Fix
Fixes #756 

## Description

The EditText in Payments QrDialog has been updated to TextInputLayout to improve userInterface

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java [hunk 1]
```diff
@@ -140,10 +140,12 @@ private void shareQr(Uri uri) {
...
 
     void showSetAmountDialog (final String qrData) {
-        final AlertDialog.Builder editTextDialog = new AlertDialog.Builder(this);
+        AlertDialog.Builder editTextDialog = new AlertDialog.Builder(this);
         editTextDialog.setCancelable(false);
         editTextDialog.setTitle("Enter Amount");
-        final EditText edittext = new EditText(this);
+        final View customLayout = getLayoutInflater().inflate(R.layout.dialog_set_amt, null);
+        editTextDialog.setView(customLayout);
+        final EditText edittext = customLayout.findViewById(R.id.editText_set_amt);
```
Comment (Bhavnaharitsa) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java:148]: State the purpose of edittext and custommlayout according to the purpose

**Comment by haran2248** [comment]
The custom layout is to set a View object to access the edittext.The editText is used to get the value amt from the layout.

**Comment by Bhavnaharitsa** [comment]
> The custom layout is to set a View object to access the edittext.The editText is used to get the value amt from the layout.

Haha, I know. I'm asking to change the variable name to something meaningful than an "edittext".

**Comment by haran2248** [comment]
> > The custom layout is to set a View object to access the edittext.The editText is used to get the value amt from the layout.
> 
> Haha, I know. I'm asking to change the variable name to something meaningful than an "edittext".

Initially it was named as editText so i didnnt want to change that

**Comment by devansh-299** [comment]
This has already been done, hence closing this PR

**Pull request closed without merging**